src/components/Timeline.js

```diff
@@ -1,10 +1,28 @@
 import React from 'react';
 import './Timeline.css';
 import TimelineEvent from './TimelineEvent';
+import PropTypes from 'prop-types';
 
-const Timeline = () => {
-  
-  return;
-}
+
+const Timeline = (props) => {
+  const eventComponents = props.events.map((event, i) => {
+    return <TimelineEvent 
+    key = {i} 
+    person = {event.person} 
+    status = {event.status}
+    timestamp = {event.timestamp} />
```

Comment:
Check the json file, small misspelling.  This is why the date's are all saying seconds ago on the timeline and why the test failed.

```suggestion
    timestamp = {event.timeStamp} />
```